test/diagnostics/Localization/no_localization_files_and_wrong_path.swift

```diff
@@ -0,0 +1,10 @@
+// RUN: %target-typecheck-verify-swift -localization-path /Not_exsisting_path -locale en
+
+// <unknown>:0: error: cannot find translations for 'en' at '/Not_exsisting_path/en.yaml': no such file
+// <unknown>:0: error: specified localization directory '/Not_exsisting_path' does not exist, translation is disabled
```

Comment:
Should this be a warning? As far as I could see in that case it would just fall back to default diagnostics?
@xedin

Reply:
Yes, it’ll fallback to default diagnostics from `.def` files

Reply:
So this probably needs to become a warning, since an error will simply not continue from here and simply fails to compile :)